File: sandbox-runner/patterns/webhooks/java/src/main/java/Shared.java
public final class Shared {
    public static final String TASK_QUEUE = "webhooks-task-queue";
    public static final String WORKFLOW_ID_PREFIX = "order";

    public record OrderInput(String orderId, double amount) {}

    public record PaymentPayload(String paymentId, double amount) {}

    private Shared() {}
}

File: sandbox-runner/patterns/webhooks/go/shared.go
package main

const (
	TaskQueue        = "webhooks-task-queue"
	WorkflowIDPrefix = "order"
	SignalName       = "payment_received"
)

type OrderInput struct {
	OrderID string  `json:"order_id"`
	Amount  float64 `json:"amount"`
}

type PaymentPayload struct {
	PaymentID string  `json:"payment_id"`
	Amount    float64 `json:"amount"`
}
